vocabularies/Common.xml

```diff
@@ -1450,6 +1450,68 @@ If the request succeeds the response will contain a JSON body of `Content-Type:
         <Annotation Term="Core.Description" String="Base URL for WebSocket connections" />
       </Term>
 
+      <TypeDefinition Name="NamespaceQualifiedActionName" UnderlyingType="Edm.String">
```

Comment:
Why namespace-qualified? CSDL XML allows to use namespace or alias as qualifier, and CSDL JSON _requires_ using the alias if one is defined.

Reply:
This is problematic as it violates CSDL rules and is inconsistent with other annotations that reference actions, for example Common.SideEffects in this vocabulary.

Reply:
Added to Albatross agenda.